electrum/plugins/satochip/README.rst

```diff
@@ -0,0 +1,83 @@
+If the wallet is not intialized yet, Electrum will perform the setup (you only need to do this once). During setup, a seed is created: this seed allows you to recover your wallet at anytime, so make sure to BACKUP THE SEED SECURELY! During setup, a PIN code is also created: this PIN allows to unlock th device to access your funds. If you try too many wrong PIN, your device will be locked indefinitely (it is 'bricked'). If you loose your PIN or brick your device, you can only recover your funds with the seed backup.
+
+The Satochip wallet is currently in Beta, use with caution!You can use the software on the Bitcoin testnet using the --testnet option.
+This software is provided 'as-is', without any express or implied warranty. In no event will the authors be held liable for any damages arising from the use of this software.
+
+Rem: Electrum uses Python 3.x. In case of error, check first that you are not trying to run Electrum with Python 2.x or with Python 2.x libraries.
+
+Development version (Windows 64bits)
+=====================================
+
+Install the latest python 3.6 release from https://www.python.org (https://www.python.org/downloads/release/python-368/)
+(Caution: installing another release than 3.6 may cause incompatibility issues with pyscard)
+
+Clone or download the code from GitHub.
+
+Open a PowerShell command line in the electrum folder
+
+In PowerShell, install the electrum dependencies::
+
+    python -m pip install .   
+    
+You may also ned to install Python3-pyqt5::
+
+    python -m pip install pyqt5
+    
+Install pyscard from https://pyscard.sourceforge.io/
+Pyscard is required to connect to the smartcard::
+
+    python -m pip install pyscard
+    
+
+In PowerShell, run electrum on the testnet (-v allows for verbose output)::
+
+    python .\run_electrum -v --testnet
+    
+
+Development version (Ubuntu)
+==============================
+(Electrum requires Python 3.6, which should be installed by default on Ubuntu)
+(If necessary, install pip: sudo apt-get install python3-pip)
+
+Electrum is a pure python application. To use the
+Qt interface, install the Qt dependencies::
+
+    sudo apt-get install python3-pyqt5
+
+Check out the code from GitHub::
+    
+    git clone git://github.com/Toporin/electrum.git
+    cd electrum
+    
+In the electrum folder:    
+    
+Run install (this should install dependencies)::
+
+    python3 -m pip install .
+    
+Install pyscard (https://pyscard.sourceforge.io/)
+Pyscard is required to connect to the smartcard:: 
+    sudo apt-get install pcscd
+    sudo apt-get install python3-pyscard
+(For alternatives, see https://github.com/LudovicRousseau/pyscard/blob/master/INSTALL.md for more detailed installation instructions)
+
+ 
+To run Electrum use::
+ python3 electrum -v --testnet 
+ 
+ 
+

```

Comment:
Not sure why this README is included? I do not see other hardware plugins having such a README.